### DISCUSSION TITLE
Search feature does not seem to work

### DISCUSSION THREAD

**Discussion by alenskas** [iterate-ch/cyberduck#14475] [Q&A]
Hello

I have had this issue for very long but now it would benefit me greatly if i can search and find file through deep folder structures in cyberduck.
I am trying to search a parent foler for all .mov files for examples so that i can view a list and download the necessary .mov files.
The search function returns 0

I have tried:
mov
.mov
*.mov
i have also tried the full and exact name of a single file

why does search not work OR is it me?

**Comment by alenskas** [comment]
FYI I am on PC and using cyberduck v8.5.7 (39535)

**Comment by alenskas** [comment]
any takers? am only i experiencing this?

**Reply by scrutinizer11** [reply]
No, you aren't alone. The feature has been broken for as long as I can remember, and by all accounts the developer doesn't care to either fix or have it removed.

**Comment by alenskas** [comment]
Any developers seeing this thread? is there something can be done to fix the search feature please?

**Comment by dkocher** [comment]
Let me know what protocol you are connecting with where the recursive search feature is broken.

**Reply by alenskas** [reply]
thank you dkocher - connecting to google drive

**Comment by dkocher** [comment]
Previously in #11424.

**Reply by alenskas** [reply]
that may be but I get 0 returns on any search I run ?

**Comment by dkocher** [comment]
We are targeting a fix in #14613 for the next release.
